### PR TITLE
New instructions for writing ledger transactions using an endorser

### PR DESCRIPTION
**Pull request by swcurran**

### CODE REVIEW

**Comment by swcurran** [comment]
This is a cleaned up version of what was there. Should be straight forward, but will take a while for you to run through.  Would like to get this posted ASAP, so if you want to just scan it, I can test and readjust as needed. 

But also would love to have your feedback... :-)

File: docs/Writing Transactions to a Ledger for an Un-privileged Author.md [hunk 1]
```diff
@@ -1,105 +1,207 @@
...
+2. The issuer extracts the ledger parts of the object from the wallet, cryptographically signs the object, and saves it to a file.
+3. The endorser accesses the signed transaction file, cryptographically signs the object, and saves it to a new file.
+4. The issuer submits the file with the doubly signed transaction and submits it to the Indy network to be written to the ledger.
+5. The Indy network verifies the signatures and the permissions of the referenced DIDs, and if all is valid, writes the transaction to the ledger.
+
+Since there are two objects, we'll go through those steps twice. At the end of the process, we'll restart the issuer agent, verify that it can start and issue a credential.
+
+## The Production Writes Workshop
+
+To get started, open a bash shell on your local system that is capable of running git and docker. If you need more information about the prerequisites, you can [learn more here]. We'll be creating three repos, so we recommend that you start in an empty directory that will be easy to clean up after the process.
```
Comment (WadeBarnes) [inline docs/Writing Transactions to a Ledger for an Un-privileged Author.md]: [learn more here] link missing

File: docs/Writing Transactions to a Ledger for an Un-privileged Author.md [hunk 2]
```diff
@@ -150,87 +250,104 @@ As the author you would send the resulting file to your endorser for signing.
   outputFile=/tmp/ian-permit.ian-co_endorser_signed_schema.txn
 ```
 
-As the endorser you would send the resulting file to the author to write to the ledger.
+**Check**: The command runs withour an error and the transaction file is created. To review it, you can run:
```
Comment (WadeBarnes) [inline docs/Writing Transactions to a Ledger for an Un-privileged Author.md]:
```suggestion
**Check**: The command runs without an error and the transaction file is created. To review it, you can run:
```

Comment (swcurran) [reply docs/Writing Transactions to a Ledger for an Un-privileged Author.md]: Fixed

File: docs/Writing Transactions to a Ledger for an Un-privileged Author.md (hunk 1 above)
Comment (swcurran) [reply docs/Writing Transactions to a Ledger for an Un-privileged Author.md]: Fixed

**Pull request merged**